test/integration/logs.test.js

```diff
@@ -0,0 +1,211 @@
+/**
+ * Copyright (c) 2017, Kinvey, Inc. All rights reserved.
+ *
+ * This software is licensed to you under the Kinvey terms of service located at
+ * http://www.kinvey.com/terms-of-use. By downloading, accessing and/or using this
+ * software, you hereby accept such terms of service  (and any agreement referenced
+ * therein) and agree that you have read, understand and agree to be bound by such
+ * terms of service and are of legal age to agree to such terms with Kinvey.
+ *
+ * This software contains valuable confidential and proprietary information of
+ * KINVEY, INC and is subject to applicable licensing agreements.
+ * Unauthorized reproduction, transmission or distribution of this file and its
+ * contents is a violation of applicable laws.
+ */
+
+const sinon = require('sinon');
+const chalk = require('chalk');
+
+const constants = require('./../../lib/constants');
+
+const command = require('./../fixtures/command.js');
+const fixtureLogs = require('./../fixtures/logs.json');
+const fixtureInternalDataLink = require('./../fixtures/kinvey-dlc.json');
+const MockServer = require('./../mock-server');
+const helper = require('./../helper');
+
+// Assumes that the output should be the logs from fixtures/logs.
+function assertLogsOutput(spy) {
+  // console.log('Query returned %s logs for FSR service %s (%s)', chalk.cyan(logs.length - skippedLogEntries.length), chalk.cyan(project.service), chalk.gray(project.serviceName));
+  const expectedSkippedLogsCount = 1;
+  const expectedLogsCount = chalk.cyan(fixtureLogs.length - expectedSkippedLogsCount);
+  const expectedId = chalk.cyan(fixtureInternalDataLink.id);
+  const expectedName = chalk.gray(fixtureInternalDataLink.name);
+  expect(spy.withArgs('Query returned %s logs for FSR service %s (%s)', expectedLogsCount, expectedId, expectedName)).to.be.calledOnce;
+
+  // console.log('%s %s - %s', chalk.green(log.containerId.substring(0, 12)), log.timestamp, chalk.cyan(messageString.trim()));
+  const expectedLogs = fixtureLogs.filter(x => x.message);
+  expectedLogs.forEach((x) => {
+    const expectedContainerId = chalk.green(x.containerId.substring(0, 12));
+    const expectedMsg = chalk.cyan(JSON.stringify(x.message));
+    expect(spy.withArgs('%s %s - %s', expectedContainerId, x.timestamp, expectedMsg)).to.be.calledOnce;
+  });
+}
+
+function buildCmdMock(start, end, pageNum, pageSize) {
+  const commandMock = {
+    opts: () => {},
+    parent: {
+      opts: () => {
+        const userOptions = {};
+        if (start) {
```

Comment:
Well, actually I find it more difficult to read when everything is on one line and since both are permitted by the style guide (https://github.com/Kinvey/style-guides/blob/master/backend/es6.md#blocks--braces) I thought I could go with multi-line blocks :fearful:

Reply:
I personally find multiple `if` statements easier to read when they're condensed. You're correct in that the style guide allows both but I still think it's important to be as consistent as possible in public-facing codebases, whether it be one way or the other. If we agree keep these `if` blocks as they are, do you think there's value splitting up other one-liners (such as in the link in my previous comment) for continuity?

Reply:
Maybe we can use both? 
(And I'll transform them to multi-lines when I take over the cli, either way :smiling_imp: :satisfied:)